docs/hermes.md

```diff
@@ -6,7 +6,7 @@ title: Using Hermes
 import M1Cocoapods from './\_markdown-m1-cocoapods.mdx';
 
 <a href="https://hermesengine.dev">
-<img width={300} height={300} style={{float: 'right', margin: '-30px 4px 0'}} src="/docs/assets/HermesLogo.svg" />
+<img width={300} height={300} className="img-inline" src="/docs/assets/HermesLogo.svg" />
```

Comment:
```suggestion
<img width={300} height={300} className="hermes-logo" src="/docs/assets/HermesLogo.svg" />
```

Let's use an unique class name, since the styling includes entries specific to the Hermes logo.